pypolygames/training.py

```diff
@@ -190,7 +190,7 @@ def get_train_reward() -> Callable[[], List[int]]:
 
         return reward
 
-    return context, assembler, get_train_reward
+    return context, assembler, get_train_reward, is_client
```

Comment:
please change the return type annotation accordingly (line 58)

Reply:
Is it correct now?